src/GraphQL/Execution/ExecutionNode.cs

```diff
@@ -1,23 +1,57 @@
 using System.Collections.Generic;
 using System.Linq;
+using GraphQL.DataLoader;
 using GraphQL.Language.AST;
 using GraphQL.Types;
 
 namespace GraphQL.Execution
 {
+    /// <summary>
+    /// Represents a node to be executed.
+    /// </summary>
     public abstract class ExecutionNode
     {
+        /// <summary>
+        /// Returns the parent node, or null if this is the root node.
+        /// </summary>
         public ExecutionNode Parent { get; }
+
+        /// <summary>
+        /// Returns the underlying graph type of this node (does not represent a <see cref="NonNullGraphType"/> or <see cref="ListGraphType"/> instance).
```

Comment:
As I can see from https://github.com/graphql-dotnet/graphql-dotnet/blob/master/src/GraphQL/Execution/ExecutionStrategy.cs#L154 it is not true.

Reply:
Correct; I will fix it.

Reply:
```suggestion
        /// Returns the underlying graph type of this node; does not represent a <see cref="NonNullGraphType"/>, and
        /// for child nodes of an array execution node, does not represent a <see cref="ListGraphType"/> instance.
```

Reply:
Please review @sungam3r